contents/tutorials/vue-surveys.md

```diff
@@ -0,0 +1,623 @@
+---
+title: How to set up surveys in Vue
+date: 2024-01-15
+author: ["lior-neu-ner"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-12.png
+tags: ['surveys']
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import EventsLight from '../images/tutorials/vue-surveys/events-light.png'
+import EventsDark from '../images/tutorials/vue-surveys/events-dark.png'
+import ImgSurveyResultsLight from '../images/tutorials/vue-surveys/survey-results-light.png'
+import ImgSurveyResultsDark from '../images/tutorials/vue-surveys/survey-results-dark.png'
+import ImgSurveyTemplatesLight from '../images/tutorials/vue-surveys/survey-templates-light.png'
+import ImgSurveyTemplatesDark from '../images/tutorials/vue-surveys/survey-templates-dark.png'
+
+[Surveys](/docs/surveys) are a great way to get feedback from your users. In this guide, we show you how to add a survey to your Vue.js app.
+
+We'll create a basic Vue app, add PostHog, create a survey, and then show you how to display the survey in the app and get responses.
+
+## 1. Create a Vue app
+
+For this tutorial, we create a basic `Vue 3` app. First, ensure [Node.js is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (version 14.6.0 or newer). Then install `@vue/cli` and create a new Vue app:
+
+```bash
+npm install -g @vue/cli
+vue create vue-surveys
+```
+
+Make sure to select `[Vue 3] babel, eslint` as the Vue version.
+
+Next, replace the code in `src/App.vue` with the following:
+
+```vue file=App.vue
+<template>
+  <div id="app">
+    <h1>This is our Vue.js survey tutorial</h1>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'App'
+}
+</script>
+```
+
+Run `npm run serve` to start your app.
+
+![Basic Vue app](../images/tutorials/vue-surveys/basic-app.png)
+
+## 2. Add PostHog
+
+> This tutorial shows how to integrate PostHog with `Vue 3`. If you're using `Vue 2`, see [our Vue docs](/docs/libraries/vue-js) for how to integrate PostHog.
+
+We use PostHog to create and control our survey as well as monitor results. If you don't have a PostHog instance, you can [sign up for free here](https://us.posthog.com/signup). 
+
+First install `posthog-js`:
+
+```bash
+npm install posthog-js
+```
+
+Create a new [plugin](https://vuejs.org/guide/reusability/plugins) by creating a new folder in your base directory called `plugins` and then a new file `posthog.js`:
+
+```bash
```

Comment:
It is slightly unclear where you are in the file system, since you don't say `cd vue-surveys` anywhere (should be earlier if included).

Reply:
I mention "in your base directory" here 😅